### PR TITLE
Acceptance test to assert regular user can be created using same email as of guest user

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance test to assert that regular user can be created using same email as of guest user

### CODE REVIEW

File: tests/acceptance/features/webUIGuests/guests.feature [hunk 1]
```diff
@@ -128,4 +128,21 @@ Feature: Guests
...
+      | user@example.com | USER@example.com | user@EXAMPLE.com |
+
+    @mailhog
+    Scenario: Guest user is not able to upload or create files
+      Given user "user0" has been created with default attributes and skeleton files
+      And the administrator has created guest user "guest" with email "guest@example.com"
+      And user "user0" has shared file "lorem.txt" with user "guest@example.com"
+      When guest user "guest" registers and sets password to "password" using the webUI
+      And user "guest@example.com" logs in using the webUI
+      Then the user should not have permission to upload or create files
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIGuests/guests.feature]: This scenario is not related to this issue

File: tests/acceptance/features/webUIGuests/guests.feature [hunk 2]
```diff
@@ -128,4 +128,21 @@ Feature: Guests
...
+
+      @mailhog
+      Scenario: Create a regular user using the same email address of an existing guest user
+        Given the administrator has created guest user "guest" with email "guest@example.com"
+        When the administrator creates a regular user with skeleton files:
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIGuests/guests.feature]: This step is not implemented

File: tests/acceptance/features/webUIGuests/guests.feature [hunk 3]
```diff
@@ -128,4 +128,12 @@ Feature: Guests
...
\ No newline at end of file
+      | user@example.com | USER@example.com | user@EXAMPLE.com |
+
+  @mailhog
+  Scenario: Create a regular user using the same email address of an existing guest user
+    Given the administrator has created guest user "guest" with email "guest@example.com"
+    When the administrator creates a regular user with skeleton files:
+      | username    | email             |
+      | regularUser | guest@example.com |
+    Then the email address of user "regularUser" should be "guest@example.com"
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIGuests/guests.feature]: This seems like api test, so it doesn't belong in the webUI features. Please move this scenario in api feature file. And create a webui step in this file.

Comment (jasson99) [reply tests/acceptance/features/webUIGuests/guests.feature]: done

**Comment by phil-davis** [comment]
We might need to run CI again tomorrow, after the core test step changes are merged and in the nightly QA tarball.

File: tests/acceptance/features/webUIGuests/guests.feature [hunk 4]
```diff
@@ -151,14 +151,25 @@ Feature: Guests
...
-    Scenario: Guest user tries to upload or create files inside the received share(read only permission)
-      Given user "user0" has been created with default attributes and skeleton files
-      And the administrator has created guest user "guest" with email "guest@example.com"
-      And user "user0" has shared folder "simple-folder" with user "guest@example.com"
-      When user "user0" updates the last share using the sharing API with
-        | permissions | read |
-      And guest user "guest" registers and sets password to "password" using the webUI
-      And user "guest@example.com" logs in using the webUI
-      And the user opens folder "simple-folder" using the webUI
-      Then the user should not have permission to upload or create files
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIGuests/guests.feature]: Why deleting this? Are you adjusting the spaces?

Comment (jasson99) [reply tests/acceptance/features/webUIGuests/guests.feature]: yes

Comment (HariBhandari07) [reply tests/acceptance/features/webUIGuests/guests.feature]: It's already merged, but I think your PR just increased the spaces in the respected scenarios.
